backend/app/services/summary_service.py

```
@@ -0,0 +1,621 @@
+            )
+            self.db.rollback()
+            try:
+                summary_data = {
+                    "status": "failed",
+                    "error": str(e),
+                    "updated_at": datetime.now().isoformat(),
+                }
+                document.summary = summary_data
+                flag_modified(document, "summary")
+                self.db.commit()
+            except Exception as commit_error:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)